src/redux/reducers/addClassReducer.js

```diff
@@ -0,0 +1,11 @@
+const setClassReducer = (state = [], action) => {
```

Comment:
Change the name of addClassReducer to setClassReducer

Reply:
done